docssrc/source/tutorials/custom_splitter/README.md

```diff
@@ -0,0 +1 @@
+Disabled unitl we can get a link to the data
```

Comment:
why would you disable the custom splitter???

Reply:
Because I don't have the data to run it, do you have those .csvs somewhere.